Documentation/gettingstarted/policy-creation.rst

```diff
@@ -0,0 +1,221 @@
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/minikube/sw_l3_l4_policy.yaml
+    ciliumnetworkpolicy.cilium.io/rule1 created
+
+Now if we run the landing requests again, we can observe in the monitor output
+that the traffic which was previously audited to be dropped by the policy are
+now reported as allowed:
+
+.. code-block:: shell-session
+
+    $ kubectl exec tiefighter -- curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
+    Ship landed
+
+Executed from the cilium pod:
+
+.. code-block:: shell-session
+
+   # cilium monitor -t policy-verdict
+   Policy verdict log: flow 0xabf3bda6 local EP ID 232, remote ID 31028, dst port 80, proto 6, ingress true, action allow, match L3-L4, 10.0.0.112 :59824 -> 10.0.0.147:80 tcp SYN
+
+Now the policy verdict states that the traffic would be allowed: ``action
+allow``. Success!
+
+Disable Policy Audit Mode
+=========================
+
+These steps should be repeated for each connection in the cluster to ensure
+that the network policy allows all of the expected traffic. The final step
+after deploying the policy is to disable Policy Audit Mode again:
+
+   .. tabs::
+
+      .. group-tab:: Configure via kubectl
+
+         .. code-block:: shell-session
+
+            $ kubectl patch -n $CILIUM_NAMESPACE configmap cilium-config --type merge --patch '{"data":{"policy-audit-mode":"false"}}'
+            configmap/cilium-config patched
+            $ kubectl -n $CILIUM_NAMESPACE rollout restart ds/cilium
+            daemonset.apps/cilium restarted
+            $ kubectl -n kube-system rollout status ds/cilium
+            Waiting for daemon set "cilium" rollout to finish: 0 of 1 updated pods are available...
+            daemon set "cilium" successfully rolled out
+
+      .. group-tab:: Helm Upgrade
+
+         .. parsed-literal::
+
+            helm upgrade cilium |CHART_RELEASE| \\
+               --namespace $CILIUM_NAMESPACE \\
+               --reuse-values \\
+               --set config.policyAuditMode=false
+
+Now if we run the landing requests again, only the *tiefighter* pods with the
+label ``org=empire`` will succeed. The *xwing* pods will be blocked!
+
+.. code-block:: shell-session
+
+    $ kubectl exec tiefighter -- curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
+    Ship landed
+
+This works as expected. Now the same request run from an *xwing* pod will fail:
+
+.. code-block:: shell-session
+
+    $ kubectl exec xwing -- curl -s -XPOST deathstar.default.svc.cluster.local/v1/request-landing
+
+This request will hang, so press Control-C to kill the curl request, or wait
+for it to time out.
+
+We hope you enjoyed the tutorial.  Feel free to play more with the setup,
+follow the `gs_http` guide, and reach out to us on the `Cilium
+Slack channel <https://cilium.herokuapp.com>`_ with any questions!
+
+Clean-up
+========
+
+.. parsed-literal::
+
+   $ kubectl delete -f \ |SCM_WEB|\/examples/minikube/http-sw-app.yaml
```

Comment:
this should work without backslashes: https://github.com/cilium/cilium/blame/1cc79c1fb522043ec8dd899285f9c0ba5cae8e88/Documentation/gettingstarted/quick-install.rst#L23

```suggestion
   $ kubectl delete -f |SCM_WEB|/examples/minikube/http-sw-app.yaml
   $ kubectl delete -f |SCM_WEB|/examples/minikube/sw_deny_policy.yaml
```

Reply:
I just copied & pasted & modified from other examples. Seems like 90% of the docs use this approach. We can follow up to make them all consistent and maybe drop the `\` if it's really not necessary.